### DISCUSSION TITLE
右侧“控制中心” 关于置顶疑问！

### DISCUSSION THREAD

**Discussion by xiamuguizhi** [ArtalkJS/Artalk#102] [Ideas]
我感觉控制中心就没必要显示置顶的评论了吧，

对应评论置顶的文章页面显示置顶评论就可以了吧，

感觉在控制中心显示置顶评论有点突兀，没必要...

![image](https://user-images.githubusercontent.com/19650297/163671871-58f3f4e1-08b6-4814-8651-faa007a98a62.png)

**Comment by qwqcode** [comment]
已更新

**Reply by xiamuguizhi** [reply]
已更新，非常感谢！

使用正常！！
